examples/observing_notebooks/AcquireAndTakeSequence.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>This is not true the way you're now launching this script (running the arun task)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>